engine/classes/Elgg/Di/ServiceProvider.php

```diff
@@ -242,6 +244,10 @@ public function __construct(\Elgg\AutoloadManager $autoload_manager) {
 		});
 
 		$this->setClassName('widgets', '\Elgg\WidgetsService');
+		
+		$this->setFactory('uploads', function(ServiceProvider $c) {
+			return $c->request->files;
```

Comment:
I'm just saying devs could just access `$services->request->files` directly. Wrapping Filebag doesn't seem necessary unless we're making a service with a bunch of useful methods around it.